src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,66 @@
+package duke.task;
+
+import duke.Parser;
+
+/**
+ * This class represents a todo task.
+ */
+public class Todo extends Task {
```

Comment:
Personally, I treat "To Do" as two words, but this is honestly nitpicky at this point; great work!